protocol/javatests/org/openyolo/protocol/HintRetrieveResultTest.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright 2017 The OpenYOLO Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.openyolo.protocol;
+
+import static org.assertj.core.api.Java6Assertions.assertThat;
+
+import android.content.Intent;
+import com.google.protobuf.ByteString;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.openyolo.protocol.Protobufs.HintRetrieveResult.ResultCode;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+@RunWith(RobolectricTestRunner.class)
+@Config(manifest = Config.NONE)
+public class HintRetrieveResultTest {
+
+    private static final String ALICE_ID = "alice@example.com";
+    private static final String ADDITIONAL_KEY = "extra";
+    private static final byte[] ADDITIONAL_VALUE = "value".getBytes();
+    private static final Map<String, byte[]> ADDITIONAL_PROPS;
+
+    private static final Hint HINT =
+            new Hint.Builder(
+                    ALICE_ID,
+                    AuthenticationMethods.EMAIL)
+                    .build();
+
+    static {
+        HashMap<String, byte[]> additionalProps = new HashMap<>();
+        additionalProps.put(ADDITIONAL_KEY, ADDITIONAL_VALUE);
+        ADDITIONAL_PROPS = Collections.unmodifiableMap(additionalProps);
+    }
+
+    @Test
+    public void build() {
```

Comment:
Please rename test methods in this file to use <method>_<qualifier>_<expected_result> format.